_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp

```diff
@@ -1549,7 +1550,9 @@ mfxStatus VAAPIEncoder::CreateAuxilliaryDevice(
 
     m_caps.ddi_caps.UserMaxFrameSizeSupport = 1;
     m_caps.ddi_caps.MBBRCSupport            = 1;
-    m_caps.ddi_caps.MbQpDataSupport         = 1;
+    // non-zero VAConfigAttribQPBlockSize means MBQP supported,
+    // VA_ATTRIB_NOT_SUPPORTED returned by legacy drivers, treat it as supported
```

Comment:
> // VA_ATTRIB_NOT_SUPPORTED returned by legacy drivers, treat it as supported

What does this mean? I don't see  you handle this here...

Reply:
`VA_ATTRIB_NOT_SUPPORTED` is non-zero, so this is supported automatically. Ok, let's do this explicit.